packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable-next-line import/no-extraneous-dependencies */
```

Comment:
Could it be better to add react-dom in peer-dependency of @talend/react-component as we use it this way ?

Reply:
You're totally right, I made the change, thanks